src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -118,7 +118,12 @@ private ViewCommandResult tagPersonModules(Model model) throws CommandException
 
         model.setPerson(personToEdit, editedPerson);
 
-        return new ViewCommandResult(MESSAGE_MODULE_TAG_PERSON_SUCCESS, editedPerson);
+        String message = MESSAGE_MODULE_TAG_PERSON_SUCCESS
+                + addedModuleTags.stream()
+                .map(ModuleTag::toString)
+                .collect(Collectors.joining("\n"));
+
+        return new ViewCommandResult(message, editedPerson);
```

Comment:
We format the `addedModuleTags` to fit within the Command Result.

Reply:
ok good job